website/content/docs/enterprise/sentinel/index.mdx

```diff
@@ -87,15 +87,31 @@ a step-by-step instruction.
 
 </Tip>
 
-Consider the following scenario. 
+<Warning>
+  Starting with Vault versions 1.15.0, 1.14.4, and 1.13.8 RGPs derived
+  from identity group memberships are applied to requests in child namespaces,
+  instead of parent ones. Read the scenarios below to familiarize yourself with the changes.
```

Comment:
```suggestion
  Starting with Vault versions 1.15.0, 1.14.4, and 1.13.8 RGPs derived
  from identity group memberships are applied to entities in child namespaces,
  only. Read the scenarios below to familiarize yourself with the changes.
```

Reply:
I believe this is correct: if I'm in the root namespace, and a child imposed a deny-all to me, I would still be able to read the child namespace, as I understand it.

Reply:
Thanks, Violet!